source/standards/storing-source-code/index.html.md.erb

```diff
@@ -6,33 +6,297 @@ old_paths:
+- A "hub and spoke" is easier to reason about and keep synchronised
+- Integrations with other tools will work with less friction, where they have a single copy to work with
+  (e.g., automated test/deployment tools, issue/bug management)
+
+### GitHub
+
+Historically, some projects use (and may remain on) private Azure DevOps and/or private GitHub repositories
+for legacy reasons, though we are now required
+to [make new source code open](https://apply-the-service-standard.education.gov.uk/service-standard/12-make-new-source-code-open.html).
+
+Specifically, we use GitHub for new and migrated work.
+
+### GitHub Organisations
+
+Department for Education (DfE) source code repositories on GitHub should be stored under an appropriate
+organisation, thereby giving appropriate oversight and protections to these source code repositories.
+
+Specifically:
+
+- The [Department for Education Digital](https://github.com/DfE-Digital)
+  GitHub organisation is used for new and existing source code repositories
+  - This is applicable to production and prototype code
+- Work created outside the DfE Digital organisation should be transferred into
+  the DfE Digital organisation at the earliest opportunity.
+  - [GitHub: Guide to transferring a repository](https://help.github.com/en/articles/transferring-a-repository)
+- The [Skills Funding Agency](https://github.com/SkillsFundingAgency/)
+  GitHub organisation is also used by the DfE
+  - Not the default for DfE-Digital
+
+If your account is added to a repository without the account being a member of the owning organisation,
+it will be counted and labelled as
+an ["outside collaborator"](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization).
+
+To join a GitHub organisation, follow the guidance and request forms available
+via [Digital Tools Support](<%= data.site.digital_tools %>).
+As there is a small cost implication for accounts to be added to a GitHub organisation,
+this should normally be done via / with support from your Delivery Manager.
+
+### Work vs Personal GitHub accounts
+	
+You may use your personal GitHub account, but you should:
+	
+- [GitHub: Add your DfE email address to your account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)
+- [GitHub: Use your secondary (DfE) email address for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)
+
+### Repository Requirements
+
+There are a number of requirements repositories in DfE must meet in order to be compliant with service standards and [Secure by Design](https://cddo.blog.gov.uk/2024/02/07/getting-started-with-the-secure-by-design-approach/) principles.
+
+Repositories must:
+
+- have branch protection turned on for the default branch in your repository either via traditional branch protection or via repository rules. Branch protection should include:
```

Comment:
I agree with this for high stakes repos (i.e., those tied to transactional services, etc) but we have loads where this is a bit overkill. I don't think this should be a hard requirement for things like prototypes and design histories.

Reply:
Changed to be explicit about which repos